dev_guide/builds.adoc

```diff
@@ -79,19 +79,19 @@ every time a Docker image tag or the source code changes:
 }
 ----
 
-<1> This specification will create a new buildConfig named ruby-sample-build.
-<2> You can specify a list of triggers, which causes a new build to be created.
-The `*ImageChange*` trigger creates a new build each time the Docker image specified in the `*strategy.from*` field changes. The `*Github*`  and `*Generic*` triggers starts the build when a callback is made from the GitHub or other source code repository hosting service, indicating  change in the source code specified in `*source*` section.
+<1> This specification will create a new `*buildConfig*` named ruby-sample-build.
+<2> You can specify a list of triggers, which cause a new build to be created.
+The `*ImageChange*` trigger creates a new build each time the Docker image specified in the `*strategy.from*` field changes. The `*github*`  and `*generic*` triggers starts the build when a callback is made from the GitHub or other source code repository hosting service, indicating change in the source code specified in the `*source*` section.
```

Comment:
Need s/starts/start/ to agree w/ plural subject, then (probably) "respectively" since `github` triggers GitHub and presumably `generic` triggers "other source code repository hosting service".